src/skin.h

```diff
@@ -127,13 +127,30 @@ class SkinInfillAreaComputation
     void generateInfill(SliceLayerPart& part, const Polygons& skin);
 
     /*!
-     * Calculate the areas which are 'directly' under air,
-     * remove them from the \ref SkinPart::inner_infill and save them in the \ref SkinPart::roofing_fill of the \p part
+     * Remove the areas which are 'directly' under air from the \ref SkinPart::inner_infill and 
+     * save them in the \ref SkinPart::roofing_fill of the \p part.
      * 
      * \param[in,out] part Where to get the sSkinParts to get the outline info from and to store the roofing areas
```

Comment:
typo, extra sSkinParts

Reply:
the typo is located on three positions line 133, line 140, line 148 probably done with a search and replace